test/generate_tests_exec_script.sh

```diff
@@ -0,0 +1,61 @@
+#!/bin/bash
+
+###################################
+# Docker image entry point script #
+###################################
+
+script_file=/tests_executor.sh
+
+mkdir -p ~/.cache
+mkdir -p ~/.local/share/keyrings
+
+# Write the tests runner script that configures DBUS and runs
+# the tests.
+cat > $script_file <<'endmsg'
+#!/bin/bash -ex
+
+eval "$(dbus-launch --sh-syntax)"
+ps -ef | grep dbus | grep -v grep
+eval "$(echo | gnome-keyring-daemon --unlock)"
+export DBUS_SESSION_BUS_ADDRESS
+export GNOME_KEYRING_CONTROL
+export SSH_AUTH_SOCK
+
+endmsg
+
+# When the tests are running against appliance the
+# tests are running using test integration executable.
+# The function builds the test integration executable command
+function _tests_runner_cmd() {
+  local _cmd="./dist/integrations_tests_runner"
+    _cmd="$_cmd --url https://$TEST_HOSTNAME"
+    _cmd="$_cmd --account $ACCOUNT"
+    _cmd="$_cmd --login $LOGIN"
+    _cmd="$_cmd --password $ADMIN_PASSWORD"
+    _cmd="$_cmd --files-folder /opt/conjur-api-python3/test"
+  echo "$_cmd"
+}
+
+# Appends the 1st argumnet to the script file
+function append_to_file() {
+  echo "$1" >> $script_file
+}
+
+if [ "$DEBUG" == "true" ]; then
+  echo "bash" >> $script_file
+elif [ "$SERVER_MODE" == "appliance" ]; then
+    # Server mode is Conjur Appliance
+    # Add tests runner executable command
+    echo "SERVER_MODE is appliance, building the test integration executable..."
```

Comment:
Maybe `Running testing against $SERVER_MODE. Building the test integration executable..

Reply:
I also wouldn't hardcode appliance for better dbugging